simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
```

Comment:
I think the name collision is not a problem, since as you mention you can rename things when you need to, or use the fully qualified name (`simpleocr.Image`) :)